app/views/enrichment_jobs/_enrichment_job.html.erb

```diff
@@ -1,179 +1,181 @@
-<%=
-  tag.div(
-    id: "enrichment-job",
-    data: {
-      url: environment_enrichment_job_path(
-        environment: params[:environment],
-        id: @enrichment_job.id,
-        format: "js"
-      ),
-      active: !@enrichment_job.finished?
-    }
-  ) do
-%>
-
-  <h2 class="left">
-    <%= t("enrichment_jobs.progress") %>
-    <% if can? :update, @enrichment_job.parser %>
-      <span class="parser-title">: <%= link_to @enrichment_job.parser.try(:name), edit_parser_path(@enrichment_job.parser) %></span>
-    <% else %>
-      <span class="parser-title">: <%= @enrichment_job.parser.try(:name) %></span>
-    <% end %>
-  </h2>
-
-  <% if @enrichment_job.status == 'active' %>
-    <%=
-      link_to(
-        t("enrichment_jobs.stop"),
-        environment_enrichment_job_path(
+<%= turbo_frame_tag dom_id(@enrichment_job) do %>
+  <%=
+    tag.div(
+      id: "enrichment-job",
+      data: {
+        url: environment_enrichment_job_path(
           environment: params[:environment],
-          id: @enrichment_job.id,
-          format: :js,
-          enrichment_job: { status: "stopped" }
+          id: @enrichment_job.id
         ),
-        method: :put,
-        remote: true,
-        id: "stop-enrichment-button",
-        class: "button new-right"
-      )
-    %>
-  <% elsif @enrichment_job.status.in?(%w[stopped failed]) %>
-    <button class="button new-right" data-open="resume-confirmation">
-      <%= t('enrichment_jobs.resume') %>
-    </button>
-
-    <%= render partial: 'shared/confirmation_modal', locals: {
-      information: t('enrichment_jobs.resume_modal_information'),
-      id: 'resume-confirmation',
-      yes:
-        button_to(
-          'Yes',
+        active: @enrichment_job.ready? || @enrichment_job.active?
+      }
+    ) do
+  %>
+
+    <h2 class="left">
+      <%= t("enrichment_jobs.progress") %>
+      <% if can? :update, @enrichment_job.parser %>
+        <%# <span class="parser-title">: <%= link_to @enrichment_job.parser.try(:name), edit_parser_path(@enrichment_job.parser)></span> %>
+        <span class="parser-title">: <%= link_to @enrichment_job.parser.try(:name), edit_parser_path(@enrichment_job.parser), target: :_top %></span>
+      <% else %>
+        <span class="parser-title">: <%= @enrichment_job.parser.try(:name) %></span>
+      <% end %>
+    </h2>
+
+    <% if @enrichment_job.status == 'active' %>
+      <%=
+        link_to(
+          t("enrichment_jobs.stop"),
```

Comment:
prefer using single quotes inside `<%= %>`

Reply:
I'll create a PR to fix all those across the project when the story is accepted. erb_lint is already on the Gemfile, I don't know why it's not part of the pipeline :thinking: